### PR TITLE
Rename Governor OZ to OZ Governor

### PR DESCRIPTION
**Pull request by prat-gpt**

### CODE REVIEW

File: lib/forge-proposal-simulator
Comment (ElliotFriedman) [inline lib/forge-proposal-simulator:1]: is this up to date on the latest version?

File: src/proposals/simple-vault-governor-oz/OZGovernorProposal_02.sol [hunk 1]
```diff
@@ -25,36 +25,36 @@ contract GovernorOZProposal_02 is GovernorOZProposal {
...
-        buildModifier(addresses.getAddress("GOVERNOR_OZ_TIMELOCK"))
+        buildModifier(addresses.getAddress("OZ_GOVERNOR_TIMELOCK"))
     {
         /// STATICCALL -- not recorded for the run stage
-        address timelock = addresses.getAddress("GOVERNOR_OZ_TIMELOCK");
-        Vault governorOZVault = Vault(addresses.getAddress("GOVERNOR_OZ_VAULT"));
-        address token = addresses.getAddress("GOVERNOR_OZ_VAULT_TOKEN");
-        (uint256 amount, ) = governorOZVault.deposits(address(token), timelock);
+        address timelock = addresses.getAddress("OZ_GOVERNOR_TIMELOCK");
+        Vault OZGovernorVault = Vault(addresses.getAddress("OZ_GOVERNOR_VAULT"));
```
Comment (ElliotFriedman) [inline src/proposals/simple-vault-governor-oz/OZGovernorProposal_02.sol]: naming: OZGovernorVault -> `ozGovernorVault`

File: test/governor-oz/OZGovernorPostProposalCheck.sol [hunk 2]
```diff
@@ -3,29 +3,29 @@ pragma solidity ^0.8.0;
...
     function setUp() public {
         string[] memory inputs = new string[](2);
         inputs[0] = "./get-latest-proposal.sh";
-        inputs[1] = "GovernorOZProposal";
+        inputs[1] = "OZGovernorProposal";
 
         string memory output = string(vm.ffi(inputs));
 
-        GovernorOZProposal governorOZproposal = GovernorOZProposal(
+        OZGovernorProposal OZGovernorproposal = OZGovernorProposal(
```
Comment (ElliotFriedman) [inline test/governor-oz/OZGovernorPostProposalCheck.sol]: camelcase variable name

File: test/governor-oz/OZGovernorVaultIntegrationTestSepolia.sol [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import {Token} from "src/mocks/vault/Token.sol";
+import {OZGovernorPostProposalCheck} from "./OZGovernorPostProposalCheck.sol";
+
+// @dev This test contract extends OZGovernorPostProposalCheck, granting it
+// the ability to interact with state modifications effected by proposals
+// and to work with newly deployed contracts, if applicable.
+contract OZGovernorVaultIntegrationTestSepolia is OZGovernorPostProposalCheck {
+    function test_addTokenToWhitelist() public {
+        Vault OZGovernorVault = Vault(addresses.getAddress("OZ_GOVERNOR_VAULT"));
+        address OZGovernorTimelock = addresses.getAddress("OZ_GOVERNOR_TIMELOCK");
```
Comment (ElliotFriedman) [inline test/governor-oz/OZGovernorVaultIntegrationTestSepolia.sol]: camelcase

File: lib/forge-proposal-simulator
Comment (prat-gpt) [reply lib/forge-proposal-simulator:1]: This is currently on the docs branch as the name of GovernorOZProposal has been updated to OZGovernorProposal on FPS. Once docs PR gets merged there I will update this to the main branch. @ElliotFriedman

**Review by ElliotFriedman — APPROVED** [review]
LGTM!

**Pull request merged**